test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/atomic/AtomicTest.scala

```diff
@@ -109,6 +111,16 @@ class AtomicTest {
     assertSame(thing2, atomic.get())
   }
 
+  @Test def atomicReferenceUpdateTest(): Unit = {
+    val atomic = new java.util.concurrent.atomic.AtomicReference(1)
+
+    val addOne: UnaryOperator[Int] = _ + 1
```

Comment:
Scala 2.11 is not happy here because it doesn't support SAMs. You'll have to spell the anonymous class out:
```suggestion
    val addOne = new UnaryOperator[Int] {
      def apply(x: Int): Int = x + 1
    }
```